libs/core/_locales/core-jsdoc-strings.json

```diff
@@ -160,7 +160,7 @@
   "rgb.setRGBLed": "Set the rgb led to a specific red, green, blue color.",
   "rgb.wheel": "Converts wheel position into an RGB color",
   "rgb.wheel|param|wheelPos": "value between 0 to 255 to get a color value, eg: 99",
-  "sensing": "Events and data from sensors\r\n\r\nControl currents in Pins for analog/digital signals, servos, i2c, ...",
+  "sensing": "Events and data from sensors\r\n\nControl currents in Pins for analog/digital signals, servos, i2c, ...",
```

Comment:
This file's changes seem largely related to converting \r\n -> \n.  This particular line went \r\n\r\n -> \r\n\n.  Is this intended?